### PR TITLE
hide-project-stats

### LINKED ISSUE #1757: Hide project stats
**Describe the new addon**

Sometimes I just don't feel like being told that a generic platformer has thousands of loves. This addon would hide the project stats on the project page (removing the numbers by the love and favorite buttons and completely removing the remix and view counts) and the counts in the What the Community is Remixing/Loving sections of the front page.

This would likely be a good addon to have an options panel for.

### PR DESCRIPTION
**Pull request by FunctionalMetatable**
Resolves #1757
### Changes

adds hide-project-stats addon

### Reason for changes

generic platformers have an average amount of views

### Tests

tested on edge

### CODE REVIEW

**Comment by apple502j** [comment]
Can't we use CSS? display: none or pointer-events and visibility should be okay

**Comment by FunctionalMetatable** [comment]
> Can't we use CSS? display: none or pointer-events and visibility should be okay

We need to keep the love and favourite buttons. The love/favourite buttons use `::before` instead of its own element

**Comment by apple502j** [comment]
At least CSS works for hiding the view and remix counts.

**Comment by apple502j** [comment]
settingsMatch should be enough IMO, not complex enough to require a ternary operator (not merged yet afaik)

**Comment by WorldLanguages** [comment]
> Requires #3060 to be merged

Why?

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,6 @@
 # Files created during packaging
 .manifest.js.bak
 .dist
+
+# vscode files
+.vscode
```
Comment (RedGuy12) [inline .gitignore:6]: does this belong here

Comment (FunctionalMetatable) [reply .gitignore:6]: just for the vscode tasks.json (I have a automated pull from upstream). should I remove it?

**Comment by FunctionalMetatable** [comment]
> settingsMatch should be enough IMO, not complex enough to require a ternary operator (not merged yet afaik)

done

**Pull request merged**